README.markdown

```diff
@@ -120,13 +125,17 @@ SignedMailBody = \
 gen_smtp_client:send({"whatever@example.com", ["andrew@hijacked.us"], SignedMailBody}, []).
 ```
 
+For using Ed25519 you need to set the option `{a, 'ed25519-sha256'}`.
+
 Don't forget to put your public key to `foo.bar._domainkey.example.com` TXT DNS record as smth like
 
 ```
+# RSA
 v=DKIM1; g=*; k=rsa; p=MIGfMA0GCSqGSIb3DQEBA......
-```
 
-See RFC6376 for more details.
+# Ed25519
+v=DKIM1; g=*; k=ed25519; p=MIGfMA0GCSqGSIb3DQEBA......
+```
```

Comment:
minor: I think it might be better to move the algorithm names outside of code blocks so it is obvious that `# RSA` is not something that should be added to the DNS record =)

---------------

Don't forget to put your public key to `foo.bar._domainkey.example.com` TXT DNS record as smth like

RSA:

```
v=DKIM1; g=*; k=rsa; p=MIGfMA0GCSqGSIb3DQEBA......
```
Ed25519:
```
v=DKIM1; g=*; k=ed25519; p=MIGfMA0GCSqGSIb3DQEBA......
```
See RFC6376 for more details.